packages/schedulers/lib/intervalScheduler.js

```diff
@@ -49,18 +61,33 @@ const init = async ({
       const lock = await redlock()
         .lock(`locks:${name}-scheduler`, 1000 * lockTtlSecs)
 
+      const extendLockInterval = setInterval(
+        async () =>
+          lock.extend(1000 * lockTtlSecs)
+            .then(() => { debug('extending lock') })
+            .catch(e => { console.warn('extending lock failed', e) }),
+        1000 * lockTtlSecs * 0.9
```

Comment:
you extend by (almost) `lockTtlSecs`, this invalidates the meaning of lock**Ttl**Secs. Maybe rename it to `oneRunTtlSecs`?